src/test/regress/expected/isolation_reference_copy_vs_all.out

```diff
@@ -227,10 +277,15 @@ count
 step s1-show-indexes: SELECT run_command_on_workers('SELECT COUNT(*) FROM pg_indexes WHERE tablename LIKE ''reference_copy%''');
 run_command_on_workers
 ---------------------------------------------------------------------
-(localhost,57637,t,1)
-(localhost,57638,t,1)
+(localhost,57637,t,2)
```

Comment:
I guess 1->2 change happens because now the shell reference table also exists on the worker?

Reply:
Yeah that is also what I thought